editor/src/components/canvas/multiplayer-cursors.tsx

```diff
@@ -305,7 +289,7 @@ const FollowingOverlay = React.memo(() => {
     }
   })
 
-  if (followed == null) {
+  if (followed == null || followedUser == null) {
```

Comment:
if `followed != null` and `followedUser == null` it would be worth writing something to the console because it should not happen